pip_pinnings.txt

```diff
@@ -1 +1 @@
-pytest<5.4
+
```

Comment:
I think this was that @astrofrog suggested on Slack?
```suggestion
pytest-doctestplus<0.6
```

Reply:
Oh wait... We need to pin both now?

Reply:
let's wait a bit more for a proper fix instead. pytest 5.4 is fine with doctestplus 0.6, but 0.5 is not working with 5.4, so we either need neither of them or both.